install.sh

```diff
@@ -51,52 +102,48 @@ function uninstall() {
         SUBCONFIGS="$(cd "$SERVICES_DIR/$SERVICE" && find . -mindepth 1 -type f)"
         for SUBCONFIG in $SUBCONFIGS ; do
             SUBCONFIG=$(sanitizePath "$SUBCONFIG")
-            echo "removing '/usr/lib/systemd/$SERVICE/$SUBCONFIG'"
-            (cd "/usr/lib/systemd/" && cd "$SERVICE" && rm -rf "$SUBCONFIG")
+            echo "removing '$DEST_DIR/lib/systemd/$SERVICE/$SUBCONFIG'"
+            rm -rf "$DEST_DIR/lib/systemd/$SERVICE/$SUBCONFIG" 2> "/dev/null" || true
         done
     done
 
-    rm "/usr/local/bin/fw-fanctrl"
-    ectool autofanctrl # restore default fan manager
-    rm "/usr/local/bin/ectool"
-    rm -rf "/etc/fw-fanctrl"
+    rm "$DEST_DIR/bin/fw-fanctrl" 2> "/dev/null" || true
+    ectool autofanctrl 2> "/dev/null" || true # restore default fan manager
+    if [ "$SHOULD_INSTALL_ECTOOL" = true ]; then
+        rm "$DEST_DIR/bin/ectool" 2> "/dev/null" || true
+    fi
+    rm -rf "$DEST_DIR$SYSCONF_DIR/fw-fanctrl" 2> "/dev/null" || true
+    rm -rf "/run/fw-fanctrl" 2> "/dev/null" || true
 
     uninstall_legacy
 }
 
-# move remaining legacy files
-function move_legacy() {
-    rm "/usr/local/bin/fanctrl.py" 2> "/dev/null" || true
-    (cp "$HOME/.config/fw-fanctrl"/* "/etc/fw-fanctrl/" && rm -rf "$HOME/.config/fw-fanctrl") 2> "/dev/null" || true
-}
-
 function install() {
-    cp "./bin/ectool" "/usr/local/bin"
-    cp "./fanctrl.py" "/usr/local/bin/fw-fanctrl"
-    chmod +x "/usr/local/bin/fw-fanctrl"
-    chown "$LOGNAME:$LOGNAME" "/usr/local/bin/fw-fanctrl"
-    mkdir -p "$HOME/.config/fw-fanctrl"
-    mkdir -p "/etc/fw-fanctrl"
+    uninstall_legacy
 
-    cp -n "./config.json" "/etc/fw-fanctrl" 2> "/dev/null" || true
+    mkdir -p "$DEST_DIR/bin"
+    if [ "$SHOULD_INSTALL_ECTOOL" = true ]; then
+        cp "./bin/ectool" "$DEST_DIR/bin/ectool"
+        chmod +x "$DEST_DIR/bin/ectool"
+    fi
+    mkdir -p "$DEST_DIR$SYSCONF_DIR/fw-fanctrl"
+    cp "./fanctrl.py" "$DEST_DIR/bin/fw-fanctrl"
```

Comment:
The path for `fw-fanctrl` and others you mention uses `DEST_DIR` as a prefix variable

Reply:
Ah, I mentioned this in a previous review, this may have gone under.

> And then make sure we use DEST_DIR + prefix for /lib and /bin consistently?

This would be the correct way. Otherwise you'd need to put /usr into DEST_DIR which also breaks config file installation again (as that turns into `/usr/etc`).

Reply:
> this may have gone under

Yes, it did